CHANGELOG.md

```diff
@@ -1,7 +1,6 @@
-## [0.29.7] 22nd May 2024
```

Comment:
why did we remove an old entry?

Reply:
sighs, when ctrl + c changes to ctrl + x, good catch tho! thanks